packages/core/src/core/index.ts

```
@@ -568,29 +664,91 @@ const ensureConnected = async (method: BaseMethod, pollingId: number) => {
+      const requestIds = requestQueue.getRequestTasksId();
+      Log.debug(
+        `Cancel Api connect requestQueues: length:${requestIds.length} requestIds:${requestIds.join(
+          ','
+        )}`
+      );
+      const canceledDevices: Device[] = [];
+      for (const requestId of requestIds) {
+        const task = requestQueue.getTask(requestId);
+        Log.debug('Cancel Api connect task: ', task);
+        if (task && task.method?.device) {
+          if (!canceledDevices.includes(task.method.device)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)